app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -207,6 +224,24 @@ public void onBackPressed() {
         }
     }
 
+    void checkKeyboardFocus(View focusedView) {
```

Comment:
Why did you add this? rebase problem?

Reply:
Settings widget has been removed. Top Bar is the upper toolbar that has only the X button at the moment.

Reply:
This rebase conflict is still not deleted